server/ethereum/index.js

```diff
@@ -31,19 +32,28 @@ const handleListenEvent = async (err, event) => {
 
   try {
     const listener = registerPinWatcher(contract)
+    // lines 36-40 will be removed
     const smartContractObj = {
       address: event.returnValues.contractAddress,
       listener
     }
     smartContracts.add(smartContractObj)
```

Comment:
This is still in here, so that i don't break all the other tests with event polling and what not.